styleguide/source/_patterns/01-molecules/08-lists/02-list-rule-metadata.twig

```diff
@@ -0,0 +1,23 @@
+{% include '07-list_title-small/list_title-small.twig' with { 'content': 'Downloads'} %}
+
+{# No metadata, list links rule #}
+
+<ul class="list_links list_links-rule">
+  <li class="list_links_item">
+    {% include '00-link.twig' with { 'content': 'Lorem ipsum dolor', 'class': 'link-black' } %}
+    {% include '10-list-link-meta.twig' with { 'content': 'PDF, 2MB' } %}
+  </li>
+  <li class="list_links_item">
+    {% include '00-link.twig' with { 'content': 'Donec non lectus leo', 'class': 'link-black' } %}
+    {% include '10-list-link-meta.twig' with { 'content': 'PDF, 5MB' } %}
+  </li>
+  <li class="list_links_item">
+    {% include '00-link.twig' with { 'content': 'Lorem ipsum dolor', 'class': 'link-black' } %}
+    {% include '10-list-link-meta.twig' with { 'content': 'PDF, 3MB' } %}
+  </li>
+  {# List Link item example with metadata #}
+  <li class="list_links_item">
+    {% include '00-link.twig' with { 'content': ' Donec non lectus leo', 'class': 'link-black' } %}
+    {% include '10-list-link-meta.twig' with { 'content': 'PDF, 2MB' } %}
```

Comment:
Hmm, good point. I'm going to do this, but when I do, since there is no easy way to include an include in the content of another include, I'll have to just use the link markup rather than the include.